marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/security.sjs

```diff
@@ -43,6 +43,10 @@ class Security {
       response.authorities.push('canManageSavedQuery');
     }
 
+    if (currentRoleNames.includes('hub-central-entity-exporter')) {
+      response.authorities.push('canExportEntityInstances');
```

Comment:
The middle tier endpoint for exporting should then have @Secured("canExportEntityInstances") on it. You can use setTestUserRoles to verify that a user with and without the role can access the endpoint.

Reply:
@akshaysonvane Can you add this in the PR https://github.com/marklogic/marklogic-data-hub/pull/3862 as you are working on adding the endpoints.

Reply:
Yes, I'll add that.